lib/paypro.js

```diff
@@ -166,8 +170,9 @@ PayPro.get = function(opts, cb) {
       memo: pd.get('memo'),
       time: pd.get('time'),
       merchant_data: md,
-      toAddress: addr.toString(),
-      amount: amount,
+      toAddress: outs[0].addr,
```

Comment:
This assumes that the first address will always be the "primary" one? Perhaps a check should be made to ensure this is selecting the address that is receiving the most? Does this `addr` need a `toString()`?

Reply:
This pull request enables multiple outputs. Previously, the first index was the only one selected. Explicitly selecting that first index for toAddress maintains backwards compatibility. A new property ("outs") has been added to this object that is an array of outputs. That is what is used in my upgrades to the wallet, but maintaining the toAddress property means that this new library is still backwards compatible with all older wallet versions.

Reply:
The toString() is now done in line 144

Reply:
The backwards compatibility angle makes sense, thanks for the explanation.